packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -65,8 +59,10 @@ const DEFAULT_VIEW = {
 	search: '',
 	page: 1,
 	perPage: 20,
+	titleField: 'title',
+	mediaField: 'preview',
 	layout: defaultLayouts[ LAYOUT_GRID ].layout,
```

Comment:
`defaultLayouts` could also be destructured after all properties and remove the `.layout`, like done in `default-views` and `post-list`.